static/js/jira-select-server.js

```diff
@@ -0,0 +1,17 @@
+$('.install-link').click(function (event) {
+  event.preventDefault();
+  const installationId = $(event.target).data('installation-id');
+  const csrfToken = document.getElementById('_csrf').value;
+  const clientKey = document.getElementById('clientKey').value;
+
+  $.post('/github/configuration', {
+    installationId,
+    _csrf: csrfToken,
+    clientKey
+  }, function (data) {
+    if (data.err) {
+      return console.log(data.err)
```

Comment:
Probably need input from jade to see what's the minimum acceptable prompt for user. UX related.

Reply:
This part will surely be revisited later, so leaving it as it is with a TODO